test/system/libs/helpers.bash

```diff
@@ -10,6 +11,11 @@ readonly IMAGE_CACHE_DIR="${BATS_RUN_TMPDIR}/image-cache"
 readonly ROOTLESS_PODMAN_STORE_DIR="${TEMP_STORAGE_DIR}/storage"
 readonly ROOTLESS_PODMAN_RUNROOT_DIR="${TEMP_STORAGE_DIR}/runroot"
 readonly PODMAN_STORE_CONFIG_FILE="${TEMP_STORAGE_DIR}/storage.conf"
+readonly DOCKER_REG_ROOT="${TEMP_STORAGE_DIR}/docker-registry-root"
```

Comment:
That brought me here, to the definition of `DOCKER_REG_ROOT`, and I saw that wiping out the `TEMP_STORAGE_DIR` solves the problem.

My question is, why are not putting `DOCKER_REG_ROOT` inside `BATS_RUN_TMPDIR` ?  Are we afraid of using tmpfs for `podman --root` ?

I think we have two options here.  Either we use a location that's unique to each run of the test suite (like `BATS_RUN_TMPDIR`), or we use `trap` to catch signals and clean everything up.  Ideally, we would do both.